frontend/src/utils/authorizations.ts

```diff
@@ -214,3 +228,57 @@ export function getMsgNameFromAuthz(authorization: any): string {
       return "Unknown";
   }
 }
+
+export interface AuthzTabs {
+  sendEnabled: boolean;
+  govEnabled: boolean;
+  stakingEnabled: boolean;
+  daosEnabled: boolean;
+  feegrantEnabled: boolean;
+  multisigEnabled: boolean;
+  authzEnabled: boolean;
+  airdropEnabled: boolean;
+}
+
+const GENERIC_AUTHZ = "/cosmos.authz.v1beta1.GenericAuthorization";
+
+export function getAuthzTabs(authorizations: any[]): AuthzTabs {
+  let result: AuthzTabs = {
+    airdropEnabled: false,
+    authzEnabled: false,
+    daosEnabled: false,
+    feegrantEnabled: false,
+    govEnabled: false,
+    multisigEnabled: false,
+    sendEnabled: false,
+    stakingEnabled: false,
+  };
+  for (let i = 0; i < authorizations.length; i++) {
+    if (authorizations[i].authorization["@type"] === SEND_V1BETA1_TYPEURL) {
+      result.sendEnabled = true;
+    } else if (authorizations[i].authorization["@type"] === GENERIC_AUTHZ) {
+      switch (authorizations[i].authorization?.msg) {
+        case SEND_V1BETA1_TYPEURL:
+          result.sendEnabled = true;
+          break;
+
+        case VOTE_V1BETA1_TYPEURL:
+          result.govEnabled = true;
+          break;
+
+        case WITHDRAW_REWARDS_V1BETA1_TYPEURL:
+        case DELEGATE_V1BETA1_TYPEURL:
+        case UNDELEGATE_V1BETA1_TYPEURL:
+        case REDELEGATE_V1BETA1_TYPEURL:
+          result.stakingEnabled = true;
+          break;
+
+        case GRANT_ALLOWANCE_V1BETA1_TYPEURL:
+        case REVOKE_ALLOWANCE_V1BETA1_TYPEURL:
```

Comment:
Also add `group` module types.

Reply:
Which group module transactions can have Authz?
I could't find it in the comos-sdk repo

Reply:
https://github.com/cosmos/cosmos-sdk/blob/v0.46.13/proto/cosmos/group/v1/tx.proto#L16

Reply:
https://github.com/vitwit/resolute/blob/c7d36a1576a556477cd763f0f1b2e5f9f0925397/frontend/src/utils/authorizations.ts#L286C1-L288C8

Reply:
`if(authorizations[i].authorization?.msg.includes("cosmos.group.")) {
        result.daosEnabled = true;
      }`